cpp/arcticdb/storage/lmdb/lmdb_storage.cpp

```diff
@@ -259,6 +259,23 @@ void LmdbStorage::do_iterate_type(KeyType key_type, const IterateTypeVisitor& vi
     }
 }
 
+bool LmdbStorage::do_is_path_valid(const std::string_view pathString) const {
+#ifdef _WIN32
+    std::string_view invalid_win32_chars = "<>:\"|?*";
```

Comment:
This list is not exhaustive:
https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names
There are also a whole bunch of other reserved filenames that would fail
https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file

Reply:
I did look at the list of other [reserved filenames](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file) while working on the fix. But the docs were strictly saying "reserved names for the **name of a file**". In fact, I tested this and I was able to create libraries with names such as `CON, PRN, COM0` which successfully created folders with these names. These are reserved names but not strictly disallowed and that's why I allowed these names.

Reply:
Aha OK that makes sense. I think it's worth adding a comment here explaining all this, as it won't be obvious to the next person reading it.